Added README.md for dummy_sensors (#573)
ros2/demos: dummy_robot/dummy_sensors/README.md

## **What Is This?**

> This demo creates and runs 2 ROS2 **nodes** individually, namely `dummy_laser` and `dummy_joint_states`.

Reviewer: ```suggestion
This demo creates and runs two ROS 2 **nodes** individually, namely `dummy_laser` and `dummy_joint_states`.
```

Author: Noted. Will edit accordingly.

> **dummy_laser** publishes `sensor_msgs::msg::LaserScan` on a ROS2 publisher topic, `/scan`.

Reviewer: ```suggestion
**dummy_laser** publishes `sensor_msgs::msg::LaserScan` on a ROS 2 publisher topic, `/scan`.
```

Author: Done.

> **dummy_joint_states** publishes `sensor_msgs::msg::JointState` on a ROS2 publisher topic, `/joint_states`.

Reviewer: ```suggestion
**dummy_joint_states** publishes `sensor_msgs::msg::JointState` on a ROS 2 publisher topic, `/joint_states`.
```

Author: Done.

> Both **dummy_laser** and **dummy_joint_states** are dependencies of **dummy_robot_bringup**. Please refer to [dummy_robot_bringup](https://github.com/ros2/demos/tree/rolling/dummy_robot/dummy_robot_bringup).

Reviewer: ```suggestion
Both **dummy_laser** and **dummy_joint_states** are dependencies of **dummy_robot_bringup**, and are usually expected to be run from the launch file there.
Please refer to [dummy_robot_bringup](https://github.com/ros2/demos/tree/rolling/dummy_robot/dummy_robot_bringup).
```

Author: Done.

## **What Is This?**

This demo creates and runs two ROS 2 **nodes** individually, namely `dummy_laser` and `dummy_joint_states`.

> **dummy_laser** publishes `sensor_msgs::msg::LaserScan` on a ROS 2 publisher topic, `/scan`.

Reviewer: ```suggestion
**dummy_laser** publishes `sensor_msgs/msg/LaserScan` on a ROS 2 publisher topic, `/scan`.
```

Author: Done.

> **dummy_joint_states** publishes `sensor_msgs::msg::JointState` on a ROS 2 publisher topic, `/joint_states`.

Reviewer: ```suggestion
**dummy_joint_states** publishes `sensor_msgs/msg/JointState` on a ROS 2 publisher topic, `/joint_states`.
```

```

### **2 - dummy_joint_states**

```bash
# Open new terminal.
ros2 topic echo /joint_states
```

```bash
# Terminal Output
---
header:
  stamp:
    sec: 1658569569
    nanosec: 414505214
  frame_id: ''
name:
- single_rrbot_joint1
- single_rrbot_joint2
position:
- -0.033653174530243454
- -0.033653174530243454
velocity: []
effort: []
---
header:
  stamp:
    sec: 1658569569
    nanosec: 434418040
  frame_id: ''
name:
- single_rrbot_joint1
- single_rrbot_joint2
position:
- -0.13326191698696957
- -0.13326191698696957
velocity: []
effort: []
---
header:
  stamp:
    sec: 1658569569
    nanosec: 454443902
  frame_id: ''
name:
- single_rrbot_joint1
- single_rrbot_joint2
position:
- -0.2315391504196944
- -0.2315391504196944
velocity: []
effort: []
---
header:
  stamp:
    sec: 1658569569
    nanosec: 474417706
  frame_id: ''
name:
- single_rrbot_joint1
- single_rrbot_joint2
position:
- -0.3275029211980919
- -0.3275029211980919
velocity: []
effort: []
---
header:
  stamp:
    sec: 1658569569
    nanosec: 494442418
  frame_id: ''
name:
- single_rrbot_joint1
- single_rrbot_joint2
position:
- -0.4201943910459491
- -0.4201943910459491
velocity: []
effort: []
---
```

## **References**

> - `sensor_msgs::msg::LaserScan` Message Format: https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/LaserScan.msg
> - `sensor_msgs::msg::JointState` Message Format: https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/JointState.msg

Reviewer: ```suggestion
- `sensor_msgs/msg/LaserScan` Message Format: https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/LaserScan.msg
- `sensor_msgs/msg/JointState` Message Format: https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/JointState.msg
```

(using slashes is kind of our language-agnostic way to refer to messages)

Author: Noted. Learnt something new today. Nice. Done.
